include/spdk_internal/lvolstore.h

```diff
@@ -115,6 +115,31 @@ struct spdk_lvol {
 	TAILQ_ENTRY(spdk_lvol)		degraded_link;
 };
 
+struct spdk_fragmap {
+	struct spdk_bit_array *map;
+
+	uint64_t cluster_size;
+	uint64_t block_size;
+	uint64_t num_clusters;
+	uint64_t num_allocated_clusters;
+};
+
+struct spdk_fragmap_req {
+	struct spdk_bdev *bdev;
+	struct spdk_bdev_desc *bdev_desc;
+	struct spdk_io_channel *bdev_io_channel;
+	//struct spdk_jsonrpc_request *request;
```

Comment:
delete this row if you think it will not be needed anymore

Reply:
Fixed. Thank you.